src/Helper/RoleHelper.js

```diff
@@ -0,0 +1,21 @@
+const config = require('config');
```

Comment:
I would suggest renaming this file to `RoleDeterminer` to clarify and limit its purpose.